frontend/src/index.css

```
@@ -3,12 +3,36 @@
 @tailwind utilities;
 
 /*
- * highlight.js の GitHub Dark テーマを import。
+ * highlight.js の GitHub Light テーマを import。
  * rehype-highlight が `<span class="hljs-keyword">` 等にトークンを分けるが、
  * 対応する CSS が無いと色付けされず読めなくなる。
- * prose のコードブロック背景は dark なので github-dark を選択。
+ * FreStyle はライトテーマ単一構成なので、 GitHub README と同じ light を採用。
  */
-@import 'highlight.js/styles/github-dark.css';
+@import 'highlight.js/styles/github.css';
```

Stylelint (17.10.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)